### DISCUSSION TITLE
Answers posted by app appear twice in app after posting

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#146853] [Mobile]
GitHub Mobile beta for Android

After I post an answer with the app, it shows twice. It doesn't appear twice in Chrome. 

![Screenshot_20241211-080330.png](https://github.com/user-attachments/assets/fc904b73-c0d8-46e8-b767-dfaeef4ea299)

![Screenshot_20241211-080338.png](https://github.com/user-attachments/assets/3ef34d9c-bfeb-4ac3-a9f1-80bda34ec648)

https://github.com/community/community/discussions/107537#discussioncomment-11536255

This happens consistently.

**Comment by taki-on** [comment]
@jsoref Thank you for reporting the issue. I will let the Android team know about the issue, and they will triage it!

**Comment by shawnz** [comment]
Any progress on this? It's been years and it still occurs regularly

**Reply by jsoref** [reply]
afaiu, it's a by-design feature. Obviously I don't like/agree with the design. If I could manage to talk to more humans who worked on the product, this should be one I'd mention. But alas, I'm not going to Universe, so ...
